wg21_paper_tracker/workspace.py

```
@@ -0,0 +1,36 @@
+_RAW_APP_SLUG = f"raw/{_APP_SLUG}"
+_MAILING_DATE_RE = re.compile(r"^\d{4}-(0[1-9]|1[0-2])$")
+
+
+def get_workspace_root() -> Path:
+    return get_workspace_path(_APP_SLUG)
+
+
+def get_raw_dir(mailing_date: str | None, year: int) -> Path:
+    """Return workspace/raw/wg21_paper_tracker/<year>/<mailing_date>/; creates if missing."""
+    if not _MAILING_DATE_RE.fullmatch(mailing_date):
+        raise ValueError("mailing_date must be in YYYY-MM format")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)